.dockerignore

```diff
@@ -0,0 +1,3 @@
+kubevirtci
```

Comment:
Is not this _kubevirtci?

Reply:
ack.

Reply:
Why do we still have kubevirtci here? https://github.com/nmstate/kubernetes-nmstate/blob/master/cluster/kubevirtci.sh#L5